### ISSUE TITLE
Subtitles with an ampersand fail to work correctly

### ISSUE THREAD

**Issue by dagwieers** [pietje666/plugin.video.vrt.nu#358]
### Describe the bug ###
I noticed this before, but failed to report it. If subtitles contain an ampersand, the next part of the line gets cut off. Example: Finale Campus Cup (episode 24) around 5:50 `CD&V` is mentioned, only `CD` is being printed.

### To Reproduce ###
1. Go to Campus Cup Finale (episode 24)
2. Forward to 5:50
3. Watch `CD&V` being cut off

### Expected behavior ###
It should show subtitles correctly.

### Additional context ###
- Operating system: LibreELEC 9.0.2
- Kodi version: 18.2
- Addon version: 2.0.0
- Using a VPN: no
- Country you are using the addon from: BE

**Comment by mediaminister** [comment]
Not a inputstream.adaptive bug. It's again a VRT NU bug.

The same bug occurs on the VRT NU-website: https://www.vrt.be/vrtnu/a-z/de-campus-cup/1/campus-cup-s1a24/

I extracted the TTML from the stream:
```
ffmpeg -i "https://remix-vrt.cdn.eurovisioncdn.net/remix/f0bf9f98-5d65-43b4-84df-d85d05a7444e/remix.ism/.mpd" -map 0:7 -c:d copy -copy_unknown -f data campuscupfinale.ttml
```
TTML Subtitle
```xml
<p begin="00:05:38.453" end="00:05:40.693" region="region-11" tts:textAlign="center">
<span style="singleHeightStyle" xml:space="preserve" tts:backgroundColor="black">Vingers aan de knoppen. Vraag 1.</span>
</p><p begin="00:05:40.773" end="00:05:46.333" region="region-10" tts:textAlign="center">
<span style="singleHeightStyle" xml:space="preserve" tts:backgroundColor="black">Belgische partijen. We kennen de </span><br></br>
<span style="singleHeightStyle" xml:space="preserve" tts:backgroundColor="black">grote partijen zoals CD</span>
</p><p begin="00:05:46.413" end="00:05:48.293" region="region-11" tts:textAlign="center">
<span style="singleHeightStyle" xml:space="preserve" tts:backgroundColor="black">Enfin, vroeger grote partijen.</span>
</p>
```
Same bug occurs with HLS and WebVTT, disabling inputstream.adaptive can't fix this:
https://remix-vrt.cdn.eurovisioncdn.net/remix/f0bf9f98-5d65-43b4-84df-d85d05a7444e/remix.ism/remix-textstream_dut=1000.vtt
```
00:05:38.453 --> 00:05:40.693
Vingers aan de knoppen. Vraag 1.

00:05:40.773 --> 00:05:46.333
Belgische partijen. We kennen de 
grote partijen zoals CD

00:05:46.413 --> 00:05:48.293
Enfin, vroeger grote partijen.
```
We can't fix a bad subtitle source.

**Comment by dagwieers** [comment]
I will report it to VRT.

**Comment by mediaminister** [comment]
I just found out that the same bug occurs with Terzake at 6:01: https://www.vrt.be/vrtnu/a-z/terzake/2019/terzake-d20190704/
Definitely a bug in VRT NU's subtitling back end.

**Comment by dagwieers** [comment]
They acknowledged the problem and forwarded it to the responsible team.

**Comment by mediaminister** [comment]
Not fixed yet: https://www.vrt.be/vrtnu/a-z/terzake/2019/terzake-d20190923/#autoplay=1054&asset=/content/dam/vrt/2019/09/23/terzake-20190923-ma-depot_WP00148139

**Comment by dagwieers** [comment]
Correct, and I asked for an update last week from VRT (with CD&V being in the news and all subtitles being cut off) but they did not have any new updates from the engineering team.

**Comment by mediaminister** [comment]
7 months later and not fixed yet: timecode 24:57 in https://www.vrt.be/vrtnu/a-z/terzake/2020/terzake-d20200129

**Comment by dagwieers** [comment]
I reported it 3 times already, and it was confirmed 3 times.
Feel free to report it once more :wink:

**Comment by dagwieers** [comment]
Still an issue, example timecode 29:55 in https://www.vrt.be/vrtnu/a-z/terzake/2020/terzake-d20200504/

**Comment by dagwieers** [comment]
@nielslaukens I reported this to VRT NU a few times the past year. Are you aware of this subtitle issue?

**Comment by nielslaukens** [comment]
We were aware of a few issues regarding subtitles, but this one wasn't explicitly mentioned. It could have been part of another issue, though. I've reached out to the corresponding team and gave this a bump, although that is not guaranteed to have any effect.

**Comment by nielslaukens** [comment]
This got escalated to our encoder vendor. They accepted this as a bug, but have not provided a timeline for fixing this. They did provide an (elaborate) workaround, which we may implement in the mean time. Keep you posted.

**Comment by mediaminister** [comment]
Just a reminder this is not fixed yet.

![Screenshot at 2020-10-28 14-33-56](https://user-images.githubusercontent.com/45148099/97442995-c1762b80-192a-11eb-937b-c66187b7bab5.png)

**Comment by michaelarnauts** [comment]
Misschien eens doorsturen naar hilde.crevits@cdenv.be dat haar partij geviseerd word door de VRT.

**Comment by dagwieers** [comment]
I reported this once more to the VRT Helpdesk.
It would be nice if we can get this fixed before the 2-year anniversary of this ticket 😉
